utils/Constants.ts

```diff
@@ -94,3 +94,16 @@ export const EXTERNAL_USD_STABLE: { [key: string]: { symbol: string; address: st
   mainnetFork: { symbol: "USDC", address: ETH_USDC_ADDRESS, decimals: 6 },
   rsk: { symbol: "XUSD", address: RSK_XUSD_ADDRESS, decimals: 18 }
 };
+
+export const BlindexFileBaseUrl = "https://blindex-static-assets.s3.filebase.com";
+export const BlindexStaticAssetsFileBaseUrl = `${BlindexFileBaseUrl}/tokens-icons`;
```

Comment:
The nr is not correct anymore.. it should be 
```suggestion
export const BlindexTokensIconsFileBaseUrl = `${BlindexFileBaseUrl}/tokens-icons`;
```